### PR TITLE
RDK-31101: Thunder changes to get SPD info

### PR DESCRIPTION
**Pull request by thanushree-r**
Reason for change: get HDMISPD data
Test Procedure: curl command to get HDMISPD data
Risks: None

Signed-off-by: thanushree.rajaselvam@sky.uk

### CODE REVIEW

File: HdmiInput/HdmiInput.cpp [hunk 1]
```diff
@@ -63,6 +65,8 @@ namespace WPEFramework
             registerMethod(HDMIINPUT_METHOD_GET_HDMI_INPUT_DEVICES, &HdmiInput::getHDMIInputDevicesWrapper, this);
             registerMethod(HDMIINPUT_METHOD_WRITE_EDID, &HdmiInput::writeEDIDWrapper, this);
             registerMethod(HDMIINPUT_METHOD_READ_EDID, &HdmiInput::readEDIDWrapper, this);
+            registerMethod(HDMIINPUT_METHOD_READ_RAWHDMISPD, &HdmiInput::getRawHDMISPDWrapper, this);
```
Comment (anand-ky) [inline HdmiInput/HdmiInput.cpp:68]: Pls increment the version and add the new APIs.

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: HdmiInput/HdmiInput.cpp (hunk 1 above)
Comment (thanushree-r) [reply HdmiInput/HdmiInput.cpp]: added changes for new api

Comment (thanushree-r) [reply HdmiInput/HdmiInput.cpp]: added changes

**Review by anand-ky — APPROVED** [review]

**Review by anand-ky — COMMENTED** [review]
Pls also cherry-pick to sprint/2103 branch.

**Pull request merged**